mindsdb/integrations/handlers/rayserve_handler/requirements.txt

```diff
@@ -0,0 +1,5 @@
+mindsdb>=22.6.2.1
```

Comment:
Let's bring the first two requirements up to the latest versions, because we need the `BaseMLEngineExec` logic, or it won't work.